core/modules/wbase/Task.cc

```diff
@@ -127,6 +130,38 @@ Task::~Task() {
 }
 
 
+std::vector<Task::Ptr> Task::createTasks(std::shared_ptr<proto::TaskMsg> const& taskMsg,
+                                         std::shared_ptr<wbase::SendChannel> sendChannel) {
+    QSERV_LOGCONTEXT_QUERY_JOB(taskMsg->queryid(), taskMsg->jobid());
+    std::vector<Task::Ptr> vect;
+
+    ///&&&; // Determine if there should be sub-tasks.
+    int fragmentCount = taskMsg->fragment_size();
+    if (fragmentCount < 1) {
+        throw Bug("QueryRunner: No fragments to execute in TaskMsg");
```

Comment:
The exception (should it be really thrown) may mislead readers that it happened in the class `QueryRunner` instead of `Task::createTasks`.